searx/engines/deepl.py

```diff
@@ -13,7 +13,7 @@
 }
 
 engine_type = 'online_dictionary'
-categories = ['general']
+categories = ['general', 'translate']
```

Comment:
To call all translators in one, I added a second category `!translate de-en hallo`